src/test/regress/expected/multi_transaction_recovery.out

```diff
@@ -371,7 +371,7 @@ COMMIT;
 SELECT COUNT(*) FROM pg_dist_transaction;
```

Comment:
The first change is definitely expected, `citus_move_shard_placement` now updates the placement metadata on all nodes and acquires locks.